pkg/ui/src/js/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load(":defs.bzl", "protobufjs_library")
+
+protobufjs_library(
+    name = "db-console_oss_js_proto",
+    out_name = "protos",
+    protos = [
+      "//pkg/server/serverpb:admin_proto",
```

Comment:
I understand you probably separated these protos out into individual targets so we could compile only the protos that HAVE to be in this JS library. Does it make a huge difference to just compile `//pkg/server/serverpb:serverpb_proto`, though? What's the impact? Code size in the generated file? Can/does that get squashed later in minification?

Reply:
Good point! The size of generated file was the primary concern for doing this but you're right, it could be reduced with minification and tree shaking of required code in generated file!